src/main/java/buildcraftAdditions/proxy/CommonProxy.java

```diff
@@ -24,7 +52,7 @@
  * http://buildcraftadditions.wordpress.com/wiki/licensing-stuff/
  */
 
-public class CommonProxy {
+public class CommonProxy implements IGuiHandler {
```

Comment:
Clean code?
The only point of this proxy class is the differentiation between server and client. A GUIHandler ineeds client/server separation. Its the perfect use of the proxy classes...